backend/src/modules/Boards/service/Boards.service.spec.js

```diff
@@ -38,4 +38,89 @@ describe('Boards service |', () => {
     //Then
     await expect(addBoardWithoutContent).rejects.toThrowError();
   });
+
+  test('when a board with the same name exist too near, error is thrown', async () => {
+    //Given
+    const boardWithTheSameName = new Board({
+      boardName: 'Wroclaw',
+      mapCoordinates: {
+        latitude: 51.88569995139321,
+        longitude: 17.02390643626451,
+      },
+      accessType: 'private',
+      adminId: '507f191e810c19729de860fa',
+      description: 'another sample description',
+      avatarUrl: 'https://firebasestorage.googleapis.com/samplenewatarurl',
+    });
+
+    //When
+    await boardsService.addBoard(testBoard);
+    const addBoardWithExistingSameTooNear = async () => {
+      await boardsService.addBoard(boardWithTheSameName);
+    };
+
+    //Then
+    await expect(addBoardWithExistingSameTooNear).rejects.toThrowError();
+  });
+});
+
+describe('Boards service | add new annoucements', () => {
+  const testAnnouncementID = '622a38a4e30a4ba39ee75a40';
+  const testBoardID = '622a38b8e7cfef8c44142cd9';
+  const inMemoryBoardsRepository = new InMemoryBoardsRepository();
+  const boardsService = new BoardsService(inMemoryBoardsRepository);
+  const testBoard = new Board({
+    boardName: 'Wroclaw',
+    mapCoordinates: { latitude: 51.88569995139321, longitude: 17.02390643626451 },
+    accessType: 'public',
+    adminId: '507f191e810c19729de860ea',
+    description: 'sample description',
+    avatarUrl: 'https://firebasestorage.googleapis.com/sampleavatarurl',
+    announcements: [],
+  });
+
+  beforeEach(() => {
+    boardsService.removeAllBoards();
```

Comment:
Jeśli metoda "removeAllBoards();" powstała tylko dla "testow" to powinna być usunięta z klasy :)))